partcad/src/partcad/part_factory_enrich.py

```
@@ -19,50 +19,62 @@
+                self.source_part_name = source_part_name.split(";")[0]
+                suffix = source_part_name.split(";")[1]
+                self.extra_with = list(map(lambda p: p.split("="), suffix.split(",")))
+            else:
+                self.source_part_name = source_part_name
+                self.extra_with = []
 
-            pc_logging.debug("Initializing an enrich to %s" % self.source)
+            self._create(config)
 
-            # TODO(clairbee): Delay de-referencing until part's initialization
+    async def instantiate(self, part):
```

Ruff (0.8.2)

`instantiate` is too complex (17 > 10)

(C901)